examples/access-controllable-examples/README.md

```diff
@@ -0,0 +1,274 @@
+use near_plugins_derive::access_control_any;
+use near_sdk::near_bindgen;
+use borsh::{BorshSerialize, BorshDeserialize};
+use near_plugins::events::AsEvent;
+use near_sdk::env;
+
+/// All types of access groups
+#[derive(AccessControlRole, Clone, Copy)]
+pub enum UsersGroups {
+    GroupA,
+    GroupB,
+}
+
+#[near_bindgen]
+#[access_control(role_type="UsersGroups")]
+#[derive(Default, BorshSerialize, BorshDeserialize)]
+struct Counter {
+    counter: u64,
+}
+
+#[near_bindgen]
+impl Counter {
+    /// In the constructor we set up a super admin, 
+    /// which can control the member lists of all user groups
+    #[init]
+    pub fn new() -> Self {
+        let mut contract: Counter = Self{
+            counter: 0,
+            __acl: __Acl::default(),
+        };
+
+        contract.__acl.init_super_admin(&near_sdk::env::predecessor_account_id());
+
+        contract
+    }
+
+    /// unprotected function, every one can call this function
+    pub fn unprotected(&mut self) {
+        self.counter += 1;
+    }
+
+    /// only the users from GroupA can call this method
+    #[access_control_any(roles(UsersGroups::GroupA))]
+    pub fn level_a_incr(&mut self) {
+        self.counter += 1;
+    }
+
+    /// only the users from GroupA or GroupB can call this method
+    #[access_control_any(roles(UsersGroups::GroupA, UsersGroups::GroupB))]
+    pub fn level_ab_incr(&mut self) {
+        self.counter += 1;
+    }
+
+
+    /// view method for get current counter value, every one can use it
+    pub fn get_counter(&self) -> u64 {
+        self.counter
+    }
+}
+```
+
+## The contract methods description
+### acl_storage_prefix
+`acl_storage_prefix` is a method that returns the common prefix of keys for storing the members and the admins of groups.
+`__acl` by default.
+
+```shell
+$ near call <CONTRACT_ACCOUNT> acl_storage_prefix --accountId <ALICE_ACCOUNT>
+Scheduling a call: <CONTRACT_ACCOUNT>.acl_storage_prefix()
+Doing account.functionCall()
+Transaction Id <TRANSACTION_ID>
+To see the transaction in the transaction explorer, please open this url in your browser
+https://explorer.testnet.near.org/transactions/<TRANSACTION_ID>
+[ 95, 95, 97, 99, 108 ]
+$ python3
+>>> print(' '.join(str(b) for b in bytes("__acl", 'utf8')))
+95 95 97 99 108
+```
+
+### acl_is_super_admin
+`acl_is_super_admin` is a _view_ method that checks that account has super admin rights.
+Super admin can control the member list of each group and control the admin list for each group.
+
+```shell
+$ near view <CONTRACT_ACCOUNT> acl_is_super_admin '{"account_id": "<SUPER_ADMIN_ACCOUNT>" }' 
+View call: <CONTRACT_ACCOUNT>.acl_is_super_admin({"account_id": "<SUPER_ADMIN_ACCOUNT>"})
+true
+```
+
+### acl_add_admin
+`acl_add_admin` is a method that adds a new admin for a specific group.
+Admins' rights don't allow running group-specific functions, but group admins can control the group member list.
+This method can be run by the admin of a specific group or by the super admin.
```

Comment:
```suggestion
This method can be run by an admin of a specific group or by a super admin.
```

There might be multiple admins and super admins.

Reply:
fixed: https://github.com/aurora-is-near/near-plugins/pull/34/commits/6fd1a4a364b2180df630ee7c4a85b3718c14c89d